client/src/services/editReview.js

```diff
@@ -0,0 +1,17 @@
+const editReview = async (reviewId, userId, content) => {
+  try {
+    const response = await fetch(`/api/v1/reviews/${reviewId}`, {
+      method: "PATCH",
+      headers: new Headers({
+        "Content-Type": "application/json",
+      }),
+      body: JSON.stringify({ reviewId, userId, content }),
```

Comment:
the reviewId is part of the params of the fetch request - the only data needed in the body is the information supplied from the user about their edits